### PR TITLE
Optimize BTC fee

### PR DESCRIPTION
**Pull request by kochol**

### CODE REVIEW

File: src/blockchain/BitcoinBlockchain.ts [hunk 1]
```diff
@@ -67,8 +69,34 @@ export class BitcoinBlockchain {
...
+                    } else if (element.maxMinutes == 180 && fee.normal == null) {
+                        fee.normal = element.minFee;
+                    } else if (element.maxMinutes == 60 && fee.high == null) {
+                        fee.high = element.minFee;
+                    }
+                });
+
+                return fee;
+            }
+            catch (error) {
```
Comment (hadideveloper) [inline src/blockchain/BitcoinBlockchain.ts:95]: Better to use MyConsole.Log(error) to capture the error message.

Comment (kochol) [reply src/blockchain/BitcoinBlockchain.ts:95]: I think there is no need to log anything here.

File: src/blockchain/BitcoinBlockchain.ts [hunk 2]
```diff
@@ -67,8 +69,34 @@ export class BitcoinBlockchain {
      * @returns BitcoinFee
      */
     public async GetTxFee(): Promise<BitcoinFee> {
-        var fees = await this._prokeyBtcBlockchain.GetTxFee();
         var fee = <BitcoinFee>{};
+        if (this._prokeyBtcBlockchain._coinName === 'BTC')
```
Comment (hadideveloper) [inline src/blockchain/BitcoinBlockchain.ts:73]: Using _coinName of _prokeyBtcBlockchain is not a good approach I think, while we have coinName in the class constructor.

Comment (kochol) [reply src/blockchain/BitcoinBlockchain.ts:73]: storing it in a new property also is not a good approach as we are taking more memory.

**Review by hadideveloper — APPROVED** [review]

**Pull request merged**